templates_ark/example_neighborhood_analysis_script.ipynb

```diff
@@ -1,18 +1,34 @@
 {
```

Comment:
<p>The notebook as it appears on GitHub has a blank cell after it, this should be removed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/714/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>